app-frontend/src/app/services/projects/project.service.js

```diff
@@ -1,5 +1,6 @@
 /* globals BUILDCONFIG L*/
```

Comment:
I would prefer these changes in the `edit.service.js` or a separate service. The project service is generally stateless and I think it would be good to keep it that way.